src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Windows.cs

```diff
@@ -942,8 +942,8 @@ public static ProcessInfo[] GetProcessInfos()
 
         private static unsafe ProcessInfo[] GetProcessInfos(IntPtr dataPtr)
         {
-            // 60 is a reasonable number for processes on a normal machine.
-            Dictionary<int, ProcessInfo> processInfos = new Dictionary<int, ProcessInfo>(60);
+            // 250 is a reasonable number for processes on a normal machine.
+            var processInfos = new List<ProcessInfo>(250);
```

Comment:
How is this change related to the rest of the PR?

It also makes the assumption that across all of the different platform implementations, we'll not get back two ProcessInfos with the same process ID; is that definitely the case?  Seems like a reasonable assumption, but calling code could break if it doesn't hold and it seems like the original author went out of his or her way to guarantee that.

Also, with regards to 60 vs 250, why did you increase that?  How did you arrive at 250 as the ideal tradeoff between allocating too large an array and having to grow the array a few times?